include/crow/http_response.h

```diff
@@ -48,6 +48,11 @@ namespace crow
             headers.emplace(std::move(key), std::move(value));
         }
 
+        std::string get_header(std::string key)
+        {
+            return headers.find(key)->second;
+        }
```

Comment:
This is redundant, you can use `crow::get_header_value(headers, key)` (the `crow::` part isn't necessary in `unittest.cpp`)

Reply:
Okay, will remove it